ui/raidboss/data/03-hw/raid/a7s.txt

```diff
@@ -3,6 +3,8 @@
 # Shasta's reddit: /u/shastaxc
 # Last Updated: 5/19/2016
 
+# Revised in http://github.com/quisquous/cactbot to use InCombat lines
```

Comment:
Some day maybe I or somebody else will fix up this timeline.  I suspect the jails are map effect lines? (Or hope?) and so maybe it's possible now.

Reply:
It's definitely a style outlier from what we have now. :-)  I'll add a todo in the file since I'm already in the sauce, and maybe someday...